runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/layouts/EventsLayout.java

```diff
@@ -110,7 +106,15 @@ private void rotateFile()
             rethrowUnchecked(ex);
         }
         buffer = createRingBuffer(path, capacity);
-        bufferSpy = createRingBufferSpy(path);
+        accessors.forEach(a -> a.setBufferSpy(createRingBufferSpy()));
```

Comment:
This doesn't seem like the right timing to move all the accessors to the latest rotated buffer.

It makes sense to rotate the events layout itself during write, but each event accessor would need to independently determine when they have reached the end of the current buffer and move to the next one (even if it's not the latest).